frontends/telegram/app/handlers/admin/subscriptions.py

```diff
@@ -0,0 +1,61 @@
+import requests
+
+from aiogram import Router, F
+from aiogram.filters import StateFilter
+from aiogram.fsm.context import FSMContext
+from aiogram.types import Message
+
+from frontends.telegram.app.states import PublishSubscription
+from frontends.telegram.app.settings import Settings
+from frontends.telegram.app.keyboards.reply.admin import main_kb
+from frontends.telegram.app.filters.role import RoleFilter
+from frontends.telegram.app.models.role import UserRole
+
+router = Router(name=__name__)
+# Apply RoleFilter to all router's handlers
+router.message.filter(RoleFilter(UserRole.ADMIN))
+router.callback_query.filter(RoleFilter(UserRole.ADMIN))
+
+
+def _api_call(method: str, params: dict) -> dict | None:
+    """
```

Comment:
Later should be migrated inside service / business logic

Reply:
No problem. When will it be migrated we can just update logic for this handler